pkg/server/server_did_configuration_test.go

```diff
@@ -20,6 +20,340 @@ import (
 	"gopkg.in/h2non/gock.v1"
 )
 
+const w3cCredentialContext = `{
```

Comment:
can this be loaded from a file instead? I thought we had context caching somewhere else...

Reply:
I wasn't aware. I found https://github.com/TBD54566975/ssi-sdk/pull/102/files , but we need to do some more piping in order to have the cache be pre-loaded. Right now, the processor caches documents, but it still needs to fetch them once from the network. 

Ok if I take on a follow up? Requires changes in SDK and the service.

Reply:
certainly, in the meantime can you move this to a file?

Reply:
Tracking in https://github.com/TBD54566975/ssi-service/issues/619